helm/prometheus-operator-app/values.yaml

```diff
@@ -1471,8 +1471,8 @@ prometheusOperator:
   ## Namespaces to scope the interaction of the Prometheus Operator and the apiserver (allow list).
   ## This is mutually exclusive with denyNamespaces. Setting this to an empty object will disable the configuration
   ##
-  namespaces: {}
-    # releaseNamespace: true
+  namespaces:
+    releaseNamespace: true
```

Comment:
Would this not cause issues with our internal use case as we check for CR in multiple namespaces ?